charts/sysdig/values.yaml

```diff
@@ -479,6 +479,19 @@ nodeAnalyzer:
       digest:
       pullPolicy: IfNotPresent
 
+    extraVolumes:
+      volumes: []
+      mounts: []
+      # example for bottlerocket
+      #volumes:
+      #  - name: socketpath
+      #    hostPath:
+      #      path: /run/dockershim.sock
+      #      type: ""
+      #mounts:
+      #  - name: socketpath
+      #    mountPath: /var/run/containerd/containerd.sock
```

Comment:
since Volumes are at pod-level and mounts are at container-level, I would prefer a slight change on the values.yaml structure.

`nodeAnalyzer.extraVolumes` could be directly the array of extra volumes (currently is `nodeAnalyzer.extraVolumes.volumes`), 
while `nodeAnalyzer.extraVolumes.mounts` should be moved into `nodeAnalyzer.runtimeScanner.extraMounts` 

makes sense to you too @dark-vex ? Same for the other chart/values.yaml

Reply:
@ProvoK yes, it makes sense, I've just looked at the extravolumes implementation made for the old image analyzer but I will change it tomorrow as per your suggestion :)

Reply:
:white_check_mark: It should be fine now! Let me know if I've missed anything :smiley: